src/hooks/use-ordered-rows.ts

```diff
@@ -14,6 +14,8 @@ export function useOrderedRows<Row>(
   rows: Row[],
   order?: Order<keyof Row>,
 ): Row[] {
+  const initialOrder = useRef(order);
```

Comment:
Hmm... this makes the hook implicitly stateful, but that isn't mentioned in the documentation. Perhaps we should keep this hook stateless and just add a `nullsLast` option to `Order` to control whether `null` values sort before or after non-null values in a field. It will be up to the caller to then set this as appropriate for each field.

Reply:
Just FYI, I changed a bit what that ref tracks after you commented here, but it still makes it stateful, so your comment is still valid.